driver/controller.go

```diff
@@ -7,13 +7,14 @@ import (
 	"strings"
 	"time"
 
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
```

Comment:
We group together 3rd party libraries. If you have no specific reason, please revert it.